### PR TITLE
Adds more accessible logging for opening harmful canisters

### PR DESCRIPTION
**Pull request by Miraviel**
## What Does This PR Do

Adds a logging entry for opening plasma/N2O canisters to the players logs (player panel -> LOGS). One could already access these via another menu but this one makes it easier to check.

No player-facing changes so I just deleted the changelog if that is ok.

## Why It's Good For The Game

Easier to find and yeet griefers.

## Images of changes

image

## Testing

1. Spawn a plasma and an N2O canister
2. Open them
3. Right-click character, player panel
4. Click on LOGS

### CODE REVIEW

File: code/modules/atmospherics/machinery/portable/canister.dm [hunk 1]
```diff
@@ -381,10 +381,12 @@ GLOBAL_DATUM_INIT(canister_icon_container, /datum/canister_icons, new())
 					if(air_contents.toxins > 0)
 						message_admins("[key_name_admin(usr)] opened a canister that contains plasma in [get_area(src)]! (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)")
 						log_admin("[key_name(usr)] opened a canister that contains plasma at [get_area(src)]: [x], [y], [z]")
+						usr.create_log(MISC_LOG, "has opened a canister of plasma")
```
Comment (Contrabang) [inline code/modules/atmospherics/machinery/portable/canister.dm:384]:
```suggestion
						ui.user.create_log(MISC_LOG, "has opened a canister of plasma")
```

File: code/modules/atmospherics/machinery/portable/canister.dm [hunk 2]
```diff
@@ -381,10 +381,12 @@ GLOBAL_DATUM_INIT(canister_icon_container, /datum/canister_icons, new())
...
 
 					if(air_contents.sleeping_agent > 0)
 						message_admins("[key_name_admin(usr)] opened a canister that contains N2O in [get_area(src)]! (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)")
 						log_admin("[key_name(usr)] opened a canister that contains N2O at [get_area(src)]: [x], [y], [z]")
+						usr.create_log(MISC_LOG, "has opened a canister of N2O")
```
Comment (Contrabang) [inline code/modules/atmospherics/machinery/portable/canister.dm:389]:
```suggestion
						ui.user.create_log(MISC_LOG, "has opened a canister of N2O")
```

File: code/modules/atmospherics/machinery/portable/canister.dm (hunk 1 above)
Comment (Sirryan2002) [reply code/modules/atmospherics/machinery/portable/canister.dm:384]: its better if this is done for the whole proc, but honestly this isn't a refactor so this is good

Comment (Miraviel) [reply code/modules/atmospherics/machinery/portable/canister.dm]: @Contrabang @Sirryan2002  Replaced all occurrences with `ui.user` and tested if it broke something, seemingly did not

**Review by Sirryan2002 — APPROVED** [review]
code looks good

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**